test/data/error/00-invalid.err

```diff
@@ -0,0 +1 @@
+00-invalid.yaml
```

Comment:
Maybe expand this a little bit more, something like `00-invalid.yaml was not found` maybe?

Reply:
Done.

I also changed the test code to catch a generic `Exception`.  Before it was specifically catching `FileNotFoundError` because it was only testing this example, but that would break as soon as we introduce another test.